### PR TITLE
[VEN-2789] Add addresses for Liquid staked ETH pool and oracles for arbitrum one and arbitrum sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
Resolves: VEN-2789

### CODE REVIEW

File: deployed-contracts/isolated-pools.md [hunk 1]
```diff
@@ -229,6 +229,19 @@
...
+* Comptroller: [`0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16`](https://arbiscan.io/address/0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16)
+* NativeTokenGateway: [`0xD1e89806BAB8Cd7680DFc7425D1fA6d7D5F0C3FE`](https://arbiscan.io/address/0xD1e89806BAB8Cd7680DFc7425D1fA6d7D5F0C3FE)
+* Underlying tokens:
+  * WSTETH: [`0x5979D7b546E38E414F7E9822514be443A4800529`](https://arbiscan.io/address/0x5979D7b546E38E414F7E9822514be443A4800529)
+  * WEETH: [`0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe`](https://arbiscan.io/address/0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe)
+  * WETH: [`0x82af49447d8a07e3bd95bd0d56f35241523fbab1`](https://arbiscan.io/address/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
+* Markets:
+  * vWSTETH\_LiquidStakedETH: [`0x9df6B5132135f14719696bBAe3C54BAb272fDb16`](https://arbiscan.io/address/0x9df6B5132135f14719696bBAe3C54BAb272fDb16)
+  * vWEETH\_LiquidStakedETH: [`0x246a35E79a3a0618535A469aDaF5091cAA9f7E88`](https://arbiscan.io/address/0x246a35E79a3a0618535A469aDaF5091cAA9f7E88)
+  * vWETH\_LiquidStakedETH: [`0x39D6d13Ea59548637104E40e729E4aABE27FE106`](https://arbiscan.io/address/0x39D6d13Ea59548637104E40e729E4aABE27FE106)
```
Comment (chechu) [inline deployed-contracts/isolated-pools.md]: To be consistent with the symbols used in other sections:

```suggestion
  * wstETH: [`0x5979D7b546E38E414F7E9822514be443A4800529`](https://arbiscan.io/address/0x5979D7b546E38E414F7E9822514be443A4800529)
  * weETH: [`0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe`](https://arbiscan.io/address/0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe)
  * WETH: [`0x82af49447d8a07e3bd95bd0d56f35241523fbab1`](https://arbiscan.io/address/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
* Markets:
  * vwstETH\_LiquidStakedETH: [`0x9df6B5132135f14719696bBAe3C54BAb272fDb16`](https://arbiscan.io/address/0x9df6B5132135f14719696bBAe3C54BAb272fDb16)
  * vweETH\_LiquidStakedETH: [`0x246a35E79a3a0618535A469aDaF5091cAA9f7E88`](https://arbiscan.io/address/0x246a35E79a3a0618535A469aDaF5091cAA9f7E88)
  * vWETH\_LiquidStakedETH: [`0x39D6d13Ea59548637104E40e729E4aABE27FE106`](https://arbiscan.io/address/0x39D6d13Ea59548637104E40e729E4aABE27FE106)
```

File: deployed-contracts/isolated-pools.md [hunk 2]
```diff
@@ -402,4 +415,17 @@
...
+* NativeTokenGateway: [`0x63cEE24b12648E36d708163587aC17a777096a47`](https://sepolia.arbiscan.io/address/0x63cEE24b12648E36d708163587aC17a777096a47)
+* Underlying tokens:
+  * WSTETH: [`0x4A9dc15aA6094eF2c7eb9d9390Ac1d71f9406fAE`](https://sepolia.arbiscan.io/address/0x4A9dc15aA6094eF2c7eb9d9390Ac1d71f9406fAE)
+  * WEETH: [`0x243141DBff86BbB0a082d790fdC21A6ff615Fa34`](https://sepolia.arbiscan.io/address/0x243141DBff86BbB0a082d790fdC21A6ff615Fa34)
+  * WETH: [`0x980B62Da83eFf3D4576C647993b0c1D7faf17c73`](https://sepolia.arbiscan.io/address/0x980B62Da83eFf3D4576C647993b0c1D7faf17c73)
+* Markets:
+  * vWSTETH\_LiquidStakedETH: [`0x253515E19e8b888a4CA5a0a3363B712402ce4046`](https://sepolia.arbiscan.io/address/0x253515E19e8b888a4CA5a0a3363B712402ce4046)
+  * vWEETH\_LiquidStakedETH: [`0x75f841b14305935D8D7E806f249D9FA52EF1550B`](https://sepolia.arbiscan.io/address/0x75f841b14305935D8D7E806f249D9FA52EF1550B)
+  * vWETH\_LiquidStakedETH: [`0xd7057250b439c0849377bB6C3263eb8f9cf49d98`](https://sepolia.arbiscan.io/address/0xd7057250b439c0849377bB6C3263eb8f9cf49d98)
```
Comment (chechu) [inline deployed-contracts/isolated-pools.md]:
```suggestion
  * wstETH: [`0x4A9dc15aA6094eF2c7eb9d9390Ac1d71f9406fAE`](https://sepolia.arbiscan.io/address/0x4A9dc15aA6094eF2c7eb9d9390Ac1d71f9406fAE)
  * weETH: [`0x243141DBff86BbB0a082d790fdC21A6ff615Fa34`](https://sepolia.arbiscan.io/address/0x243141DBff86BbB0a082d790fdC21A6ff615Fa34)
  * WETH: [`0x980B62Da83eFf3D4576C647993b0c1D7faf17c73`](https://sepolia.arbiscan.io/address/0x980B62Da83eFf3D4576C647993b0c1D7faf17c73)
* Markets:
  * vwstETH\_LiquidStakedETH: [`0x253515E19e8b888a4CA5a0a3363B712402ce4046`](https://sepolia.arbiscan.io/address/0x253515E19e8b888a4CA5a0a3363B712402ce4046)
  * vweETH\_LiquidStakedETH: [`0x75f841b14305935D8D7E806f249D9FA52EF1550B`](https://sepolia.arbiscan.io/address/0x75f841b14305935D8D7E806f249D9FA52EF1550B)
  * vWETH\_LiquidStakedETH: [`0xd7057250b439c0849377bB6C3263eb8f9cf49d98`](https://sepolia.arbiscan.io/address/0xd7057250b439c0849377bB6C3263eb8f9cf49d98)
```

File: risk/resilient-price-oracle.md [hunk 3]
```diff
@@ -165,13 +165,16 @@ The current list of correlated token oracles in Venus is:
...
+
 | Pool | Market | MAIN oracle | PIVOT oracle | FALLBACK oracle | Notes |
 |---|---|---|---|---|---|
 | Core | WBTC | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
 | Core | WETH | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
 | Core | USDC | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
 | Core | USDT | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
 | Core | ARB | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
+| Core | WEETH |  [weETHOneJumpChainlinkOracle](https://arbiscan.io/address/0x09eA4825a5e2FB2CB9a44F317c22e7D65053ea9d) | - | - | |
+| Core | WSTETH |  [wstETHOneJumpChainlinkOracle](https://arbiscan.io/address/0x748DeFdbaE5215cdF0C436c538804823b55D76bc) | - | - | |
```
Comment (chechu) [inline risk/resilient-price-oracle.md]:
```suggestion
| Liquid Staked ETH | WEETH |  [weETHOneJumpChainlinkOracle](https://arbiscan.io/address/0x09eA4825a5e2FB2CB9a44F317c22e7D65053ea9d) | - | - | |
| Liquid Staked ETH | WSTETH |  [wstETHOneJumpChainlinkOracle](https://arbiscan.io/address/0x748DeFdbaE5215cdF0C436c538804823b55D76bc) | - | - | |
| Liquid Staked ETH | WETH | [SequencerChainlinkOracle](https://arbiscan.io/address/0x9cd9fcc7e3deda360de7c080590aad377ac9f113) | - | - | |
```

File: deployed-contracts/isolated-pools.md (hunk 1 above)
Comment (Debugger022) [reply deployed-contracts/isolated-pools.md]: Fixed, 8db135cb50f38b062efa6eb42b239f48186bb652

File: deployed-contracts/isolated-pools.md (hunk 2 above)
Comment (Debugger022) [reply deployed-contracts/isolated-pools.md]: Fixed, 8db135cb50f38b062efa6eb42b239f48186bb652

File: risk/resilient-price-oracle.md (hunk 3 above)
Comment (Debugger022) [reply risk/resilient-price-oracle.md]: Added, 8db135cb50f38b062efa6eb42b239f48186bb652

**Review by chechu — APPROVED** [review]

**Pull request merged**